### ISSUE TITLE
Issue with sample script

### ISSUE THREAD

**Issue by jamesorlando** [MicrosoftDocs/azure-stack-docs#905]
The sample PowerShell script for updating the MySQL RP has some kind of format/character issue. You can tell by the different color of -DefaultSSLCertificatePassword parameter (line 40) . If you copy and past it to PowerShell ISE that too shows a different color. You can resolve the issue by removing the backtick( ` ) at the end of the -AzureEnvironment $AzureEnvironment line 39. 

If you run the scrip as is it will start, but you will be prompted for every parameter, even if you already filled it out (and accept the EULA). You will ultimately get an error and the deployment will fail due to a failure to deploy the MySQLAdapter. An Azure Stack Operator will be able to verify the failure is due to "Deployment template validation failed: 'The provided value for the template parameter 'setupPfxPassword' at line '121' and column '29' is not valid.'." 

I had a successful upgrade by collapsing the parameters into a single line and was not prompted for any further input. 

![image](https://user-images.githubusercontent.com/68352831/87594530-56218a00-c6aa-11ea-8332-ebf56370bbf2.png)
![image](https://user-images.githubusercontent.com/68352831/87594618-7c472a00-c6aa-11ea-8345-a53f5988e125.png)
![image](https://user-images.githubusercontent.com/68352831/87594762-b0bae600-c6aa-11ea-8be1-aa7d52f5923a.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4b377dc6-ab2b-9f37-41a6-c0142bc0529a
* Version Independent ID: 71f3378d-6db4-6941-4514-7ff63a9c5adc
* Content: [Update the MySQL resource provider in Azure Stack Hub - Azure Stack Hub](https://docs.microsoft.com/en-us/azure-stack/operator/azure-stack-mysql-resource-provider-update?view=azs-2002)
* Content Source: [azure-stack/operator/azure-stack-mysql-resource-provider-update.md](https://github.com/MicrosoftDocs/azure-stack-docs/blob/master/azure-stack/operator/azure-stack-mysql-resource-provider-update.md)
* Service: **azure-stack**
* GitHub Login: @BryanLa
* Microsoft Alias: **bryanla**

**Comment by BryanLa** [comment]
@jamesorlando - thanks very much for your contribution, in spotting and reporting this. 

It appears to have been caused by the fact that ```$PfxPass ` ``` was missing a trailing space, after the backtick character. I've tested in our test environment. I have not tested the execution of the script, but it it resolves the color formatting issue. I'll be redeploying the revised article at 3pm Pacific today.
